Algorithm.CSharp/SectorExposureRiskFrameworkAlgorithm.cs

```diff
@@ -38,7 +38,7 @@ public override void Initialize()
             // Set requested data resolution
             UniverseSettings.Resolution = Resolution.Daily;
 
-            SetStartDate(2014, 03, 24);
+            SetStartDate(2014, 03, 25);
```

Comment:
Is this change due to only certain data being available locally?

Reply:
Exactly 👍